### PR TITLE
Run Git-related tests from the ATH in the pipeline

### PR DESCRIPTION
**Pull request by recampbell**

### CODE REVIEW

File: Jenkinsfile [hunk 1]
```diff
@@ -50,3 +53,13 @@ void mvn(def args) {
     }
   }
 }
+
+void runAthForPlugin(String plugin, String coreVersion, String testPattern) {
+  dir ('acceptance-test-harness') {
+    git 'git@github.com:jenkinsci/acceptance-test-harness.git'
```
Comment (rtyler) [inline Jenkinsfile]: This should use `git://` since our build agents will not have SSH keys for GitHub access configured

Comment (recampbell) [inline Jenkinsfile]: Fixed

**Comment by andresrc** [comment]
:bee:

**Comment by recampbell** [comment]
Noting that this depends on https://github.com/jenkinsci/acceptance-test-harness/pull/190 to fix the test failures.

**Comment by recampbell** [comment]
Ready to merge after jenkinsci/acceptance-test-harness#190

**Comment by recampbell** [comment]
@rtyler I've tried a couple of times, but I can't get the build to trigger on ci.jenkins.io. Halp?

**Comment by rtyler** [comment]
@recampbell I think the GitHub Org Folder doesn't appreciate the Close/Reopen rebuild trick as whatever plugin runs on `jenkins.ci.c.c`. I have manually rebuilt, one option in the future might be to simply rebase against HEAD of master

**Comment by MarkEWaite** [comment]
@recampbell I'm going to close this pull request for now.

I believe we need significantly more work on the git related contents of the ATH before we run them in the git plugin CI environment.

I need to close or merge pull requests for the plugin so that we have far fewer than 50 pull requests being evaluated by each run of the CI environment.

The ATH git tests need to be confirmed that they are reliable and consistent.

The ATH git tests need to be visited with a critical eye to decide which tests are better covered in a unit test or an integration test, and which truly require the end to end testing provided by the acceptance test harness.

**Pull request closed without merging**